### DISCUSSION TITLE
Add option to reset weather source instances to default

### DISCUSSION THREAD

**Discussion by min7-i** [breezy-weather/breezy-weather#1488] [Ideas]
### Describe your idea

While it's possible to set a custom instance for weather sources in settings, there is currently no direct way to reset an instance to the default. The only way is to manually input the default instance again.

Therefore, the idea is to provide a new option in the edit dialog to reset the instance.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Ah, I was pretty sure it existed. It's fixed, and even avoids an issue where the user would input the exact same URL, and if that URL changed in the future, the user would be stuck with the old URL he inputted manually.

Clearing the field will show as a placeholder the default value now that can be saved to be reset.

**Reply by min7-i** [reply]
Great. Thanks!
